example.js

```diff
@@ -237,12 +237,43 @@ client.on('message', async msg => {
         const newStatus = msg.body.split(' ')[1];
         await client.setStatus(newStatus);
         msg.reply(`Status was updated to *${newStatus}*`);
-    } else if (msg.body === '!mention') {
-        const contact = await msg.getContact();
+    } else if (msg.body === '!mentionUsers') {
         const chat = await msg.getChat();
-        chat.sendMessage(`Hi @${contact.number}!`, {
-            mentions: [contact]
+        // To mention one user you can pass the ID as is without wrapping it in Array:
+        await chat.sendMessage('Hi @number!', {
+            mentions: 'number@c.us'
         });
+        // To mention a list of users:
+        await chat.sendMessage('Hi @number1, @number2!', {
+            mentions: ['number1@c.us', 'number2@c.us']
```

Comment:
This is not example.
Please use:
```js
chat.sendMessage(`Hi @${contact.number}!`, {
            mentions: [contact]
        });
```

Reply:
In the last wweb versions you don't have to pass a contact object to mentions, you can simply pass a serialized id like this:
```js
clietn.sendMessage(groupId, 'test, @xxxxxxxxxxxxx', { mentions: ['xxxxxxxxxxxxx@c.us'] });
```

Reply:
I focus on the fact that 'number1@c.us' its not a real number